packages/vscode-extension/src/debug/depsChecker/doctorConstant.ts

```diff
@@ -1,12 +1,20 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+const InstallNode =
+  "Go to https://nodejs.org/en/about/releases/ to install Node.js (recommended version v14)";
```

Comment:
https://nodejs.org/about/releases/
(Removing the `en` part)

Reply:
Fixed.